kroxylicious-operator/src/main/java/io/kroxylicious/kubernetes/operator/KafkaProxyIngressStatusFactory.java

```diff
@@ -51,13 +61,20 @@ KafkaProxyIngress newFalseConditionStatusPatch(KafkaProxyIngress observedProxy,
                                                    String reason,
                                                    String message) {
         Condition falseCondition = newFalseCondition(observedProxy, type, reason, message);
-        return ingressStatusPatch(observedProxy, falseCondition);
+        return ingressStatusPatch(observedProxy, falseCondition, MetadataChecksumGenerator.NO_CHECKSUM_SPECIFIED);
     }
 
     @Override
     KafkaProxyIngress newTrueConditionStatusPatch(KafkaProxyIngress observedProxy,
-                                                  Condition.Type type) {
+                                                  Condition.Type type, String checksum) {
```

Comment:
I'm not sure what our policy is on single arg per line, but where we have used it already, we ought to be consistent.